homeassistant/components/update/__init__.py

```diff
@@ -276,6 +291,12 @@ async def async_skip(self) -> None:
         self.__skipped_version = latest_version
         self.async_write_ha_state()
 
+    @final
+    async def async_clear_skipped(self) -> None:
+        """Clear the skipped version."""
+        self.__skipped_version = None
+        self.async_write_ha_state()
```

Comment:
These are not needed if the entity is not skipped.
Should we guard that to avoid writing an un-needed state?

Reply:
It should not trigger an actual state change, if the state is unchanged.